config/sarc-dev.json

```diff
@@ -78,7 +141,27 @@
             "prometheus_url": null,
             "prometheus_headers_file": null,
             "start_date": "2022-04-01",
-            "nodes_info_file": "secrets/nodes_cedar.txt"
+            "nodes_info_file": "secrets/nodes_cedar.txt",
+            "gpus_per_nodes": {
+                "cdr{{[26-386]}}": {
+                    "p100": "NVIDIA P100-12G PCIe"
+                },
+                "cdr{{[876-904]}}": {
+                    "p100l": "NVIDIA P100-16G PCIe"
+                },
+                "cdr{{[905-906]}}": {
+                    "p100": "NVIDIA P100-12G PCIe"
+                },
+                "cdr{{[908-911]}}": {
+                    "p100l": "NVIDIA P100-16G PCIe"
+                },
+                "cdr{{[912-922]}}": {
+                    "p100": "NVIDIA P100-12G PCIe"
+                },
+                "cdr{{[2468-2678]}}": {
+                    "v100l": "NVIDIA V100-32G PCIe"
+                }
+            }
```

Comment:
I guessed that all of the GPUs where PCIe connected? The info doesn't seams to be present in the docs : https://docs.alliancecan.ca/wiki/Cedar. Maybe it's cleaner to simply omit the socket for the non v100 GPUs?